pages/common/dotnet.md

```diff
@@ -0,0 +1,19 @@
+# dotnet
+
+> .NET Command Line Tools
+
+- Initialize a new .NET project
```

Comment:
Don't know what's up with Travis CI here! Anyway, we prefer a colon at the end of each command description.